homeassistant/components/mazda/__init__.py

```diff
@@ -102,6 +102,13 @@ async def async_handle_service_call(service_call: ServiceCall) -> None:
         if vehicle_id == 0 or api_client is None:
             raise HomeAssistantError("Vehicle ID not found")
 
+        if service_call.service != "send_poi":
+            _LOGGER.warning(
+                "The mazda.%s service is deprecated and has been replaced by a button entity; "
```

Comment:
Question for reviewers: How long should I wait before removing the deprecated services? I looked through the developer docs but didn't see any guidelines for how long the deprecation period should be.

Reply:
I'm not a member but I can tell you with certainty that the deprecation period is at least 2 release cycles.